go/libkb/login_state.go

```diff
@@ -200,7 +211,10 @@ func (s *LoginState) GetPassphraseStream(ui SecretUI) (pps *PassphraseStream, er
 	if pps != nil {
 		return pps, nil
 	}
-	if err = s.verifyPassphraseWithServer(ui); err != nil {
```

Comment:
I follow now.  It's doing the same thing, as you said above.  Ok, this is really a minimal change then.